Documentation/Signoffs/VelocityAndAcceleration.md

```diff
@@ -0,0 +1,126 @@
+| 5 | The design shall have a mimumum range for projectile detection of 45 inches from the point of placement. | This constraint arises from the range of detection for the projectile and time to obtain appropriate data within the time window. |
+
+</div>
+
+### Further Explanation
+[3] The 100ms input delay is the time that was allocated for detection within the critical path time of 1.78 seconds.
+
+[5] The constraint stated to have a minimum range of 45 inches. This is the maximim distance from the middle sensor post to the edge of the game board and this is the minimum distance away from the edge of the gameboard that the sensor would be loacated at. The measurments for the game board and competition rules can be found [10]. 
+
+
+## Schematics
+<p align = "center">
+<img src = https://github.com/JTJones73/Capstone2024-Team2/blob/awmorrow42-signoff-SensorPost-VelocityAndAcceleration/Documentation/Images/VelocityAndAccelerationSchematic-1.png width='400'>
+</p>
+
+## Analysis
+### Velocity and Acceleration Measurements
+This device will fuction in multiple parts. One part is having two servo's for aiming the ToF sensor up/down and left/right based of the path  detected by the projectile path sensing subsystem. The resoning for this is due to the sensor having a feild of view of 25 degrees[3] so the sensor needs to be aimed in the general direction of the projectile for detection. Using the set path we can keep the sensor aimed in the projectiles direction as needed. After the mesurements are taken the sensor will be returned to its default position and await the next detected path. This will allow the device to operate without any input or intervention from user.
+The next part of this subsystem is the velocity and acceleration calculations. The formula for Velocity is 
+
+$$ Velocity = {\Delta Distance \over \Delta Time} $$
+
+Where distance is the distance travled divided by the time it took to go that distance. Using this formula the following can be obtained
+
+$$ Velocity = { |d_1 - d_2| \over |t_1 - t_2|}$$
+
+In this formula the values for d will be the distance measurements from the time of flight sensors, and t will be the time stamps taken when those measurements are saved. Using the found velocities the acceleration of the projectile can be calculated by the following formula.
+
+$$ Acceleration = {\Delta Velocity \over \Delta Time} $$
+
+Breaking the formula down further 
+
+$$ Acceleration = { |v_1 - v_2| \over |t_1 - t_2|} $$
+
+The values for v will be the saved velocities calculated before and t will be the corilating times saved when calculated. 
+Some notes about this approch it is assumed that the inital velocity of the projectile is 0 or close to it and is under a constant acceleration. These assumptions will allow for simpler calculations and may be adjusted for futher accuracy if needed.
```

Comment:
Your calculations assume that the ball is traveling strait at the distance sensor. In reality the ball is traveling "across" the sensor. You do not have to show me now but you will have to take this into account in the calculations.